demos/tests/cases.py

```diff
@@ -173,6 +173,24 @@ def single_option_cases(key, *args):
             ModelArg('instance-segmentation-security-0091')),
     )),
 
+    CppDemo(name='smart_framing_demo', implementation='cpp_gapi',
+            model_keys=['-m_yolo', '-m_sr'],
+            device_keys=['-d_yolo', '-d_sr'],
+            test_cases=combine_cases(
+        TestCase(options={'--no_show': None, '-at_sr': '3ch',
+            **MONITORS,
+            '-i': DataPatternArg('smart-classroom-demo'),
```

Comment:
smart_framing_demo

Reply:
I don't have smart framing specific data. So I reused smart classroom data

Reply:
Ok. I will add comment about it bit later.